config/default-spinnaker-local.yml

```diff
@@ -110,9 +109,11 @@ providers:
     enabled: ${SPINNAKER_KUBERNETES_ENABLED:false}
 
     primaryCredentials:
-      name: my-docker-registry-account
-      address: https://index.docker.io/
-      repository: library/nginx
+      name: my-docker-registry
```

Comment:
I removed `account`, since it's not an account like the others (you don't deploy to your docker account), instead it's `registry` in the sense that it's used to pull stored images out of. Since `primaryCredentials` is singular, I don't think the name needs to reflect the specific registry being used. Otherwise `my-kubernetes-account` should include the name of the cluster in use, and `my-google-account` should include the name of the project.

I'll link to the configuration docs in the comment above.